### ISSUE TITLE
Linux support

### ISSUE THREAD

**Issue by martinsmaslovs** [tsaost/autoload-temporary-addon#1]
Does this work on linux as well?

**Comment by IgitBuh** [comment]
Yes, it does. I just got it working for the Firefox docker container (which is of course Linux based).

You simply put the files in
```
usr/lib/firefox/userChrome.js
/usr/lib/firefox/defaults/pref/config-prefs.js
```

Make sure to edit the `userChrome.js` and in `installUnpackedExtensions()` remove the example paths and leave only one  `installExtension` line with the path to your own add-on file (e.g. in zip format).

**In Docker, I have done this:**

To make maintaining easy, I created the folder `debug_files` in `docker/Firefox` and placed all required files (`userChrome.js`, `config-prefs.js`, `addon.zip`) inside it.

Now I can map the two *.js files to the required locations and the `addon.zip` file is simply available through the internal `/config` path.

To summarize:

Volume mapping:
```
docker/Firefox  ->  /config   (this is useful regardless of this hack for accessing config and download files)
docker/Firefox/debug_files/userChrome.js  ->  usr/lib/firefox/userChrome.js
docker/Firefox/debug_files/config-prefs.js  ->  /usr/lib/firefox/defaults/pref/config-prefs.js
```

In `userChrome.js` modify `installUnpackedExtensions()` like this:
```
nstallExtension("/config/debug_files/addon.zip", true);
```

**Comment by syntaxfiend** [comment]
I'm trying this out now. But I've installed firefox via snap. and so I think the path that I need to add things to is `/snap/firefox/usr/lib/firefox`

**Comment by syntaxfiend** [comment]
I got this to work with ubuntu 22.04 installed from snap by:
- adding userChrome.js to `/etc/firefox/userChrome.js` 
- and config-prefs.js to `/etc/firefox/defaults/pref/config-prefs.js`
- apply the changes described in the PR for ff version 117
